src/cli/backup/exchange.go

```diff
@@ -284,31 +284,29 @@ func createExchangeCmd(cmd *cobra.Command, args []string) error {
 
 	for _, sel := range sel.SplitByResourceOwner(users) {
```

Comment:
this is variable shadowing and can easily lead to errors. It should be changed to have a different variable name

Reply:
I'm pretty sure this is safe.  But no harm in changing it.  Better readability if nothing else.